src/test/ui/error-codes/E0275.stderr

```diff
@@ -9,132 +9,7 @@ LL | impl<T> Foo for T where Bar<T>: Foo {}
    |
    = help: consider adding a `#![recursion_limit="256"]` attribute to your crate (`E0275`)
    = note: required because of the requirements on the impl of `Foo` for `Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<Bar<T>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>`
```

Comment:
Should we show one more item here?

Reply:
Seems fine to me either way, r=me with this fix or without

Reply:
Given the nature of these diagnostics, each line is super long.

Reply:
True, I wonder if we can truncate the first and second line to the terminal width? It would be good if we can dump the real thing somewhere.
```
Bar<Bar<...<Bar>>>
```
But I imagine this hueristic might not always work or might be hard?

Reply:
That heuristic _might_ break down in some case that I'm not anticipating. Changing the pretty printer to do this will likely be involved too, but is likely worth it and could be used in other places, I just don't have the energy to tackle that now 😅